plugins/webp-uploads/hooks.php

```diff
@@ -764,6 +770,10 @@ function webp_uploads_modify_webp_quality( int $quality, string $mime_type ): in
 function webp_uploads_render_generator(): void {
 	// Use the plugin slug as it is immutable.
 	echo '<meta name="generator" content="webp-uploads ' . esc_attr( WEBP_UPLOADS_VERSION ) . '">' . "\n";
+
+	if ( webp_uploads_is_fallback_all_sizes_enabled() ) {
+		echo '<meta name="generator" content="webp-uploads-fallback-all-sizes ' . esc_attr( WEBP_UPLOADS_VERSION ) . '">' . "\n";
+	}
```

Comment:
Why is that being added? We shouldn't have an extra generator tag for this feature specifically. If we need feature-specific indicators in the frontend, let's discuss that in a separate issue.

Reply:
In the #1513 there is mention of the adding the generator tag thats why I added it.
> Ideally when users enable this feature we should notate that in the Generator tag so we can later discover how many sites opt into generating all image sizes.

Should I remove the generator tag code from this PR?

Reply:
Yeah, I think so. Maybe worth opening an issue to discuss whether/how we want to make more specific features discoverable in the frontend? Such an issue could link to that comment as one example reference.

Reply:
Removed the generator tag code from this PR.